tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -4874,19 +4874,31 @@ public function propfindResultShouldContainEntries(
 		$elementRows = $expectedFiles->getRows();
 		$should = ($shouldOrNot !== "not");
 		foreach ($elementRows as $expectedFile) {
-			$fileFound = $this->findEntryFromPropfindResponse(
-				$expectedFile[0],
-				$user,
-				$method,
-				"files",
-				$folderpath
-			);
+			if ($expectedFile[0] === '') {
```

Comment:
save and use it in other places
```suggestion
                        $resource = $expectedFile[0];
			if ($resource === '') {
```